hasher-matcher-actioner/terraform/main.tf

```diff
@@ -265,4 +267,17 @@ module "actions" {
   log_retention_in_days = var.log_retention_in_days
   additional_tags       = merge(var.additional_tags, local.common_tags)
   measure_performance   = var.measure_performance
+  api_token_secret      = aws_secretsmanager_secret.api_token
+}
+
+### ThreatExchange API Token Secret ###
+
+resource "aws_secretsmanager_secret" "api_token" {
```

Comment:
If this is specifically "threatexchange" api_token, can we scope the name as well? "threatexchange_api_token"?